ui/Main.elm

```diff
@@ -75,93 +69,11 @@ update msg ({ filterValues } as model) =
                         NewBuildIdSearch value ->
                             { model | buildIdFilter = value }
 
-                updatedFilteredBuilds =
-                    applyFilters updatedModelWithFilters
+                updatedRoute =
```

Comment:
Nit: this is a `Model` value, `updatedRoute` sounds strange. `updatedModel` maybe?